clean_project/project/tests/data/config.cfg

```diff
@@ -0,0 +1,7 @@
+[db_connection]
+database = indexdb
+user = gianluca
+password = password
+host = indexdb.c0awqz4pgr31.us-west-2.rds.amazonaws.com
```

Comment:
This is a personal file for testing. In the package will add config/config.cfg file